pkg/api/file_repository.go

```diff
@@ -36,21 +41,40 @@ func NewFileSystemRepository(dir string) (*FileSystemRepository, error) {
 			}
 
 			definition := repo.parseDefinition(appConfigBody)
-			if err = repo.Add(definition); err != nil {
-				log.WithError(err).Error("Can't add the definition to the repository")
-				return nil, err
+			for _, v := range definition.defs {
+				if err = repo.Add(&v); err != nil {
+					log.WithError(err).Error("Can't add the definition" + v.Name + " to the repository")
```

Comment:
Let's use pkg FTW `fmt.Sprintf("Can't add the definition %s to the repository", v.Name)`

Reply:
Ah actually we can do even better
```go
log.WithField("name", v.Name).WithError(err).Error("Failed during add definition to the repository")
```

Reply:
Nice, I'll change it as soon as I can